buildpatches/com_github_awslabs_soci_snapshotter.patch

```diff
@@ -0,0 +1,34 @@
+diff --git a/MODULE.bazel b/MODULE.bazel
+new file mode 100644
+index 0000000..7ab2087
+--- /dev/null
++++ b/MODULE.bazel
+@@ -0,0 +1,6 @@
++module(
++    name = "soci_snapshotter",
++    version = "0.0.5",
```

Comment:
I just bumped this to 0.0.7, do you mind updating it here as well?

Reply:
done!